java/client/src/main/java/glide/managers/CommandManager.java

```diff
@@ -218,6 +220,26 @@ public <T> CompletableFuture<T> submitClusterScan(
         return submitCommandToChannel(command, responseHandler);
     }
 
+    /**
+     * Submit a password update request to GLIDE core.
+     *
+     * @param password A new password to set or empty value to remove the password.
+     * @param mode Password update mode.
+     * @param responseHandler A response handler
+     * @return A request promise.
+     * @param <T> Type of the response.
+     */
+    public <T> CompletableFuture<T> submitPasswordUpdate(
+            Optional<String> password,
```

Comment:
Should this be NonNull, like how it is in BaseClient?

Reply:
I think it is enough to check it on one place.